x-pack/metricbeat/module/azure/compute_vm/compute_vm_test.go

```diff
@@ -63,6 +63,9 @@ func TestFetch(t *testing.T) {
 		t.Fatal(err)
 	}
 	module, metricsets, err = mb.NewModule(c, mb.Registry)
+	if err != nil {
```

Comment:
How about using `assert.Error()` here?

Reply:
assert.Error asserts that a function returned an error but in this case we do not expect one.

Reply:
Yes that case we can use assert. NoError. But anyway, using `if err!=nil` here works too.